about.html

```diff
@@ -455,8 +452,7 @@ <h3>Elisabeth Chin</h3>
     </div>
     <div id="harish-krishnan-bio" class="col-md-7 col-md-pull-5 col-sm-7 col-sm-pull-5 col-xs-12">
       <h3>Harish Krishnan</h3>
-      <p>Harish Krishnan is a Masters of Business Analytics 2020 candidate at University of British Columbia and a Supply chain professional with over five years experience solving business and process problems using data analytics and continious
-        improvement tools.</p>
+      <p>Harish Krishnan is the Director of the Director, Centre for Operations Excellence and a Professor of Operations and Logistics at UBC Sauder School of Business. He specializes in Contracts and supply chain coordination, Incentive distortions in supply chains and supply chain management.</p>
```

Comment:
I don't think Harish is the COE director anymore, but he is the Program Director for MBAN? Also I would suggest keeping the capitalization of phrases at the end of this sentence consistent.